README.md

```diff
@@ -53,3 +55,33 @@ If you are interested in doing a translation, please follow these steps:
 1. We will merge your translation!
 1. HTML and plain text versions are automatically generated from your markdown file.
 1. A link to your translation is automatically added to the translations page.
+
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/EthicalSource/ethicalsource.dev/graphs/contributors"><img src="https://opencollective.com/ethical-source/contributors.svg?width=890&button=false" /></a>
+
+### Financial Contributors
+
+Become a financial contributor and help us sustain our community. [[Contribute](https://opencollective.com/ethical-source/contribute)]
```

Comment:
Once this gets merged, we should specify what donations are used for. Not a blocker tho.